x-pack/plugins/apm/public/components/app/TransactionDetails/index.tsx

```diff
@@ -49,6 +53,10 @@ export function TransactionDetails() {
       <EuiPanel>
         <TransactionDistribution
           distribution={distributionData}
+          loading={
+            distributionStatus === FETCH_STATUS.LOADING ||
+            distributionStatus === undefined
```

Comment:
I understand that comparing the `status` value from `useTransactionDetailsCharts` with the `FETCH_STATUS.LOADING` constant from `useFetcher` is correct, but it feels cumbersome and leaks implementation details of `useTransactionDetailsCharts`. Perhaps `useTransactionDistribution` could explicitly expose the boolean flag `isLoading` to achieve the same result?

Reply:
sounds good, but it seems like the status is explicitly exposed from the hook anyway (and from `useFetcher` calls in general). In the interest of time, I'll merge this one as is (and backport it to 7.{2,x}), we can improve on this later  (maybe it should be solved in `useFetcher`).